### PR TITLE
Delet group api integration

### PR DESCRIPTION
**Pull request by pran-sada**
### Description

- Delete group API integration

#### Video

https://user-images.githubusercontent.com/116863369/205031294-9d8d47bb-dc0b-4c5c-b88a-23b704cb5e8c.mp4

### Other Details

- [JIRA Ticket](https://collectiveshift.atlassian.net/browse/SNHUT-2736)

#### Checklist

- [ ] **Assumptions of User Story met (including acceptance criteria)**
- [ ] **All new necessary Unit tests were CREATED** to promote code coverage
- [ ] **All required unit tests are PASSING**
- [ ] The proposed functionality was tested manually locally, and it's working fine.
- [ ] PR was properly reviewed by a team member (at least 1 approval required to merge!)
- [ ] Unnecessary console.logs AND/OR comments were removed
- [ ] Feature ok-ed by UX designer (If needed - not for back end components) and Product Owner
- [ ] Ensure configuration and/or build changes documented

### CODE REVIEW

File: apps/toboggan-app/src/app/group/components/group-list/group-list.component.ts [hunk 1]
```diff
@@ -212,7 +198,29 @@ export class GroupListComponent implements OnInit, OnDestroy {
...
+    try {
+      this.modalAlertService.hideModalAlert();
+      await this.deleteGroupAPI(uuid);
+      this.showNotification(
+        'success',
+        ``,
+        `The <strong>${name}</strong> user group has been deleted.`,
+        true
+      );
+      console.log('333');
```
Comment (sada-ajith) [inline apps/toboggan-app/src/app/group/components/group-list/group-list.component.ts:211]: remove this console

File: apps/toboggan-app/src/app/group/components/group-list/group-list.component.ts [hunk 2]
```diff
@@ -212,7 +198,29 @@ export class GroupListComponent implements OnInit, OnDestroy {
...
+        'error',
+        `Delete group`,
+        `couldn't be completed.`,
+        true,
+        null
+      );
+    }
+  }
+
+  private async deleteGroupAPI(id: string) {
```
Comment (sada-ajith) [inline apps/toboggan-app/src/app/group/components/group-list/group-list.component.ts:223]: rename id to uuid

File: apps/toboggan-app/src/app/group/services/group.service.ts [hunk 3]
```diff
@@ -36,8 +36,8 @@ export class GroupService {
   }
 
   //Delete group
-  async deleteGroup(groupId: string) {
-    await firstValueFrom(this.http.delete(`/api/groups/${groupId}`));
+  async deleteGroup(id: string) {
```
Comment (sada-ajith) [inline apps/toboggan-app/src/app/group/services/group.service.ts:39]: rename id to uuid

**Review by sada-ajith — CHANGES_REQUESTED** [review]
Could you please check review comments and make necessary changes

File: apps/toboggan-app/src/app/group/components/group-list/group-list.component.ts (hunk 1 above)
Comment (pran-sada) [reply apps/toboggan-app/src/app/group/components/group-list/group-list.component.ts]: Removed

File: apps/toboggan-app/src/app/group/services/group.service.ts (hunk 3 above)
Comment (pran-sada) [reply apps/toboggan-app/src/app/group/services/group.service.ts]: Changed to uuid

**Review by sada-ajith — APPROVED** [review]
Approved

**Pull request merged**